documentation/week3/src/components/List.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import Task from './Task';
+
+export default class List extends Component {
+   
+    render() {
+        
+        return (
+            <div className='List'>
```

Comment:
Don't use capital letters `List` with classes. only small letters, but if you want to add many words just combine them with `-` or `_`